src/web/theme/pages/TryIt/Step/Step.scss

```diff
@@ -0,0 +1,33 @@
+.step {
+  &__title {
+    $this: &;
+    padding: $tryItPadding;
+    font-weight: $bold;
+    padding: 2rem;
+    font-weight: 700;
+    display: flex;
+    align-items: baseline;
+
+    & > * {
+      margin-right: 1em;
+    }
+
+    &__pins,
+    #{$this}--validated &__pins {
```

Comment:
I know that I suggested to try something like this but when looking at it now it doesn't seem any clearer than repeating the full selector :laughing:

Reply:
Maybe I wrote it the wrong way ?